db/migrate/20170601062052_change_column_of_requests.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeColumnOfRequests < ActiveRecord::Migration[5.0]
+  def change
+    change_column :requests, :status, 'integer USING CAST(status AS integer)'
```

Comment:
what is that ?
Where did you find this?

The migration should be like this: 
`change_column :requests, :status, :integer, default: 0`
As simple as that.

But I'll be curious to know where you found the above line.